### PR TITLE
HTML5 vocabulary with CssDefaults term

### PR DESCRIPTION
**Pull request by BerSie**
new HTML5 vocabulary, new CssDefaults term with property "width"; term applies to record;
UI vocabulary adapted that this term is a valid term for UI.DataFieldAbstract

### CODE REVIEW

File: vocabularies/HTML5.json [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+            ],
+            "@Common.Experimental": true,
+            "@Core.Description": "CSS definitions for e.g. UI.DataFieldAbstract annotations"
+        },
+        "CssDefaultsType": {
+            "$Kind": "ComplexType",
+            "@Common.Experimental": true,
+            "width": {
+                "$Nullable": true,
+                "@Core.Description": "css: width"
```
Comment (erlethor) [inline vocabularies/HTML5.json:55]: Shall we refere somehow to the spec? https://www.w3.org/TR/CSS21/visudet.html#propdef-width

And I would add to the description something like.

`The property allows all values specified for the  original css width property. Note that any client consuming this annotation may only support selected length units`

File: vocabularies/HTML5.json [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+            }
+        ],
+        "CssDefaults": {
+            "$Kind": "Term",
+            "$Type": "HTML5.CssDefaultsType",
+            "$AppliesTo": [
+                "Record"
+            ],
+            "@Common.Experimental": true,
+            "@Core.Description": "CSS definitions for e.g. UI.DataFieldAbstract annotations"
```
Comment (erlethor) [inline vocabularies/HTML5.json:48]: Should we point from HTML5 to UI? 

Or could we just live with `Selected CSS properties that may be used as defaults by HTLM5 clients`

File: vocabularies/UI.json [hunk 3]
```diff
@@ -1609,7 +1617,8 @@
             "@Core.Description": "Elementary building block that represents a piece of data and/or allows triggering an action",
             "@Validation.ApplicableTerms": [
                 "UI.Hidden",
-                "UI.Importance"
+                "UI.Importance",
+                "HTML5.CssDefaults"
```
Comment (erlethor) [inline vocabularies/UI.json:1621]: If we remove the "e.g. in UI.DataFieldAbstract" in HTML5 Vocab. We may add a comment to the long description here

File: README.md [hunk 4]
```diff
@@ -17,6 +17,7 @@ This repository defines the following SAP vocabularies:
 * [PersonalData: Personal data / GDPR](vocabularies/PersonalData.md)
 * [Session: Sticky Sessions](vocabularies/Session.md)
 * [UI: Representing data in user interfaces](vocabularies/UI.md)
+* [UI5: Rendering directives for UI5](vocabularies/UI5.md)
```
[the remark this answers is not in the document]
Comment (BerSie) [reply README.md]: Done

File: vocabularies/HTML5.md [hunk 5]
```diff
@@ -0,0 +1,20 @@
+# HTML5 Vocabulary
+**Namespace: [com.sap.vocabularies.HTML5.v1](HTML5.xml)**
+
+Terms for HTML5
+
+The HTML5 Vocabulary provides rendering hints for HTML5 clients
```
[the remark this answers is not in the document]
Comment (BerSie) [reply vocabularies/HTML5.md]: Done

File: vocabularies/HTML5.json (hunk 1 above)
Comment (BerSie) [reply vocabularies/HTML5.json]: Added the reference and description

File: vocabularies/HTML5.json (hunk 2 above)
Comment (BerSie) [reply vocabularies/HTML5.json]: This reference is OK imo (I don't like the other way round). I took you proposal to enhance the description

File: vocabularies/UI.json (hunk 3 above)
Comment (BerSie) [reply vocabularies/UI.json:1621]: I was not sure where to put here a long description - maybe to all the applicable terms?

Comment (erlethor) [reply vocabularies/UI.json:1621]: I meant in the LongDescription of the DataFieldAbstract above (not visible in the comment)

Comment (BerSie) [reply vocabularies/UI.json:1622]: OK - I made a proposal

**Pull request merged**